frontends/web/src/components/header/Header.css

```diff
@@ -76,54 +76,6 @@
     width: 20px;
 }
 
-.guideWrapper {
-    position: fixed;
-    top: 0;
-    right: 0;
-    width: 56px;
-    height: 56px;
-    z-index: 4003;
-    /* overflow: hidden; */
-}
-
-.guideToggler {
-    position: relative;
-    top: -48px;
-    right: -21px;
-    color: var(--color-white);
-    background-color: var(--color-darkblue);
-    width: 96px;
-    height: 96px;
-    transform: rotate(45deg);
-    font-size: var(--size-medium);
-    transition: all ease 0.1s;
-}
-
-.guideWrapper:hover .guideToggler {
-    top: -48px;
-    right: -10px;
-}
-
-.guideToggler span {
-    position: absolute;
-    bottom: 5px;
-    left: 28px;
-    cursor: default;
-    display: block;
-    width: 18px;
-    height: 18px;
-    line-height: 18px;
-    text-align: center;
-    transform: rotate(-45deg);
-    transition: all ease 0.1s;
-    user-select: none;
-}
-
-.guideWrapper:hover .guideToggler span {
```

Comment:
Just looking at the code (without checking it out), didn't the hovering get lost? 🤔 Please ignore if not. 🙂

Reply:
@KasparEtter -- yeah, playing around with it for a while and first thought the animation didn't really make the usability better so I removed it, but I put it back now with some adjustments.